### ISSUE TITLE
App Crash upon go to setting a download. 

### ISSUE THREAD

**Issue by BTheBlood** [tachiyomiorg/tachiyomi#8309]
### Steps to reproduce

Open app > go to setting > download > crash 

### Expected behavior

Should be fine like before update.

### Actual behavior

Every thing is fine and can download normally but not setting a download option. 

### Crash logs

content://eu.kanade.tachiyomi.provider/ext_cache_files/tachiyomi_crash_logs.txt

### Tachiyomi version

0.14.0

### Android version

Androids 8.0.1

### Device

Samsung j7 2016

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.14.0](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.
[tachiyomi_crash_logs.txt](https://github.com/tachiyomiorg/tachiyomi/files/9892412/tachiyomi_crash_logs.txt)

**Comment by curche** [comment]
You actually haven't uploaded any crashlogs
you can create a new crashlog and send it here (or the old one is still in your notifications)

**Comment by arkon** [comment]
Please upload the actual crash log file.

**Comment by BTheBlood** [comment]
[tachiyomi_crash_logs.txt](https://github.com/tachiyomiorg/tachiyomi/files/9892417/tachiyomi_crash_logs.txt)
Sorry I kinda confused here hope it will work.

**Comment by stevenyomi** [comment]
It's probably a broken translation string. What language are you using?

**Comment by arkon** [comment]
It doesn't seem to be language-specific looking at other crash logs.

I think it's because you end up with paths like

```
content://com.android.externalstorage.documents/tree/2292-0911%3A/document/2292-0911%3A
```

Which is [attempted to be formatted](https://github.com/tachiyomiorg/tachiyomi/blob/master/app/src/main/java/eu/kanade/presentation/more/settings/widget/ListPreferenceWidget.kt#L42), but the `%3A` part isn't actually supposed to be formatted with something.

**Comment by arkon** [comment]
Fixed by #8322

**Comment by BTheBlood** [comment]
[Uploading tachiyomi_crash_logs.txt…]()
